### PR TITLE
Handles both Python 2 & 3, and multiple optimisations & simplifications.

### PR DESCRIPTION
**Pull request by BertrandBordage**

### CODE REVIEW

File: src/wagtail_personalisation/admin.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 from django.contrib import admin
 
-from wagtail_personalisation import models, rules
+from . import models, rules
```
Comment (mvantellingen) [inline src/wagtail_personalisation/admin.py]: I'm not really a fan of relative imports, it always requires me to visualise the directory tree to see where files are imported from. Can we keep them absolute?

File: src/wagtail_personalisation/admin_urls.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 from django.conf.urls import url
 
-from wagtail_personalisation import views
+from . import views
```
Comment (mvantellingen) [inline src/wagtail_personalisation/admin_urls.py]: See above

File: src/wagtail_personalisation/blocks.py [hunk 3]
```diff
@@ -3,13 +3,13 @@
 from django.utils.translation import ugettext_lazy as _
 from wagtail.wagtailcore import blocks
 
-from wagtail_personalisation.adapters import get_segment_adapter
-from wagtail_personalisation.models import Segment
+from .adapters import get_segment_adapter
+from .models import Segment
```
Comment (mvantellingen) [inline src/wagtail_personalisation/blocks.py]: See above please.

File: src/wagtail_personalisation/models.py [hunk 4]
```diff
@@ -11,9 +11,9 @@
     FieldPanel, FieldRowPanel, InlinePanel, MultiFieldPanel, ObjectList,
     PageChooserPanel, TabbedInterface)
 
-from wagtail_personalisation.forms import AdminPersonalisablePageForm
-from wagtail_personalisation.rules import AbstractBaseRule
-from wagtail_personalisation.utils import count_active_days
+from .forms import AdminPersonalisablePageForm
+from .rules import AbstractBaseRule
+from .utils import count_active_days
```
Comment (mvantellingen) [inline src/wagtail_personalisation/models.py]: Also above

File: src/wagtail_personalisation/wagtail_hooks.py [hunk 5]
```diff
@@ -3,18 +3,17 @@
...
-from wagtail.wagtailcore.models import Page
 
-from wagtail_personalisation import admin_urls
-from wagtail_personalisation.adapters import get_segment_adapter
-from wagtail_personalisation.models import PersonalisablePageMixin, Segment
-from wagtail_personalisation.utils import impersonate_other_page
+from . import admin_urls
+from .adapters import get_segment_adapter
+from .models import PersonalisablePageMixin, Segment
+from .utils import impersonate_other_page
```
Comment (mvantellingen) [inline src/wagtail_personalisation/wagtail_hooks.py]: Absolute please

**Comment by BertrandBordage** [comment]
Removed the relative imports.

**Pull request merged**